versioned_docs/version-8.0/apis-clients/java-client/index.md

```diff
@@ -86,6 +86,10 @@ ZeebeClient client =
         .build();
 ```
 
+## Javadoc
+
+The official Java client library API documentation can be found [here](https://javadoc.io/doc/io.camunda/zeebe-client-java). These are standard Javadocs, so your favorite JVM IDE will be able to install them locally as well
```

Comment:
🔧 We can link directly to the versioned API. 

Note that there is an automatic re-direction to the latest patch of a minor when using the minor version in the URL, i.e. https://javadoc.io/doc/io.camunda/zeebe-client-java/8.0 is currently redirected to https://javadoc.io/doc/io.camunda/zeebe-client-java/8.0.8/index.html